core/domains/email_manager.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2018 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Config functions for managing email notifications."""
+
+from core import feconf
+from core.platform import gae_email_services
+from google.appengine.api import app_identity
+
+
+def send_mail_to_admin(email_subject, email_body):
+    """Send an email to the admin email address.
+
+    The email is sent to the ADMIN_EMAIL_ADDRESS set in feconf.py.
+
+    Args:
+        email_subject: str. Subject of the email.
+        email_body: str. Body (message) of the email.
+    """
+
+    app_id = app_identity.get_application_id()
+    body = '(Sent from %s)\n\n%s' % (app_id, email_body)
```

Comment:
For this use case, let's set the reply-to correctly and not change the body, so that if the admin receives the email and replies, it looks just like a regular reply to the original contact "email" that was submitted through the form.

Reply:
Wouldn't it be better for the admin to forward the auto-mail from Foundation page to reply? 
When the admin reply, they would be replying to Foundation page's system email. Would this mean that we need a handler for incoming mail as well?

Reply:
It should be possible to set the Reply-To field on an email so that when the admin clicks Reply, the recipient email address will be set correctly. See e.g. the "Reply-To" entry here: http://www.gnu.org/software/emacs/manual/html_node/emacs/Mail-Headers.html

Reply:
Ah gotcha ... reply_to field added to gae_email_service